conn.go

```diff
@@ -278,6 +278,54 @@ func (c *Conn) handleGreet(enhanced bool, arg string) {
 	c.writeResponse(250, NoEnhancedCode, args...)
 }
 
+func findSharpBrackets(arg string) (string, string, error) {
+	if arg[0] != '<' {
+		return "", "", errors.New("must start with <")
+	}
+	for i := 1; i < len(arg); i++ {
+		if arg[i] == '>' {
+			return arg[0 : i+1], arg[i+1:], nil
+		}
+		if arg[i] == '<' {
+			sharp, _, err := findSharpBrackets(arg[i:])
+			if err != nil {
+				return "", "", err
+			}
+			// -1 because the for loop will increment i
+			i += len(sharp) - 1
+		}
+	}
+	return "", "", errors.New("must end with >")
+}
+
+func getFromAndArgs(arg string) (from string, args []string, err error) {
+	arg = strings.TrimSpace(arg)
+	args = strings.Split(arg, " ")
```

Comment:
This line can be removed, it's unused in the `if` block and is already in the `else` block.